src/test/resources/features/Message.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Message
+  In order to use the app
+  As a user
+  I must be able to send and receive messages
+
+  Scenario: Send message and logged
+    Given I login as "demo" with password "password"
+    And The message with id 1 doesn't exist
```

Comment:
As you don't have control over the identifiers assigned to resources, it is better to check here that there are no messages in the database. You can use the count method in the repository and assert that it equals 0